src/EventStore.Core/TransactionLog/Chunks/TFChunk/TFChunk.cs

```diff
@@ -99,7 +100,7 @@ public enum ChunkVersions : byte {
 		// the lock protects all three parts of the caching process:
 		// CacheInMemory, UnCacheFromMemory and TryDestructMemStreams
 		// so that the variables _cacheStatus, _cachedData, and _cachedLength are synchronized by the lock,
-		// and so is the creation and removal of the mem readers.
+		// and so is the creation and removal of the pooled mem readers.
```

Comment:
Is this an intended change? looks like it may be an old change from the base branch: `timothycoleman/chunk-lock`

Reply:
ah yeah we can remove the word pooled here, the lock guards them all (pooled or not). i'll do that